commitizen/cz/base.py

```diff
@@ -17,6 +33,10 @@ def questions(self) -> list:
     def message(self, answers: dict) -> str:
         """Format your git message."""
 
+    @property
+    def style(self):
+        return merge_styles(BaseCommitizen.DEFAULT_STYLE_CONFIG, Style(self.config["style"]))
```

Comment:
This line fails, I think it should be something like this instead:

```python
@property
def style(self):
    return merge_styles(
        [Style(BaseCommitizen.DEFAULT_STYLE_CONFIG), Style(self.config["style"])]
    )


```

Reply:
Yes, I've fixed it. Thanks!